api/resources/views/vendor/mail/html/themes/default.css

```diff
@@ -0,0 +1,299 @@
+/* Base */
```

Comment:
Do we need to export all those blade files? If there is an update, we'll need to update all manually.
Nothing is changed I guess to this file